src/twisted/internet/base.py

```diff
@@ -75,24 +111,25 @@ def __init__(self, time, func, args, kw, cancel, reset, seconds=runtimeSeconds):
         self.canceller = cancel
         self.seconds = seconds
         self.cancelled = self.called = 0
-        self.delayed_time = 0
+        self.delayed_time = 0.0
```

Comment:
Note that there are many places in `twisted.internet` where time is represented by "a number" and others where it it more clearly a `float` and others where it's documented as an `int`.

I'm trying to consistently use `float`.

Reply:
👍 , `float` is the correct type.

Reply:
Cool thanks for confirming. I was pretty sure but not 100%